Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM tomcat:8-jre8
+MAINTAINER Alexandros Sigaras <als2076@med.cornell.edu>, Fedde Schaeffer <fedde@thehyve.nl>
+LABEL Description="cBioPortal for Cancer Genomics"
+ENV APP_NAME="cbioportal" \
+    PORTAL_HOME="/cbioportal"
+#======== Install Prerequisites ===============#
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        git \
+        libmysql-java \
+        patch \
+        python \
+        python-jinja2 \
+        python-mysqldb \
+        python-requests \
+    # install new versions of these packages backported to Debian stable;
+    # Debian does not add new features or break backwards compatibility within
+    # a stable release, but for these dependencies we need versions that do.
+    && apt-get install -y --no-install-recommends -t jessie-backports \
+        maven \
+        openjdk-8-jdk \
+    && ln -s /usr/share/java/mysql-connector-java.jar "$CATALINA_HOME"/lib/ \
+    && rm -rf $CATALINA_HOME/webapps/examples \
+    && rm -rf /var/lib/apt/lists/*
+#======== Configure cBioPortal ===========================#
+COPY . $PORTAL_HOME
+WORKDIR $PORTAL_HOME
+EXPOSE 8080
+#======== Build cBioPortal on Startup ===============#
+CMD mvn -DskipTests clean install \
+     && cp $PORTAL_HOME/portal/target/cbioportal*.war $CATALINA_HOME/webapps/cbioportal.war \
+     && find $PWD/core/src/main/scripts/ -type f -executable \! -name '*.pl'  -print0 | xargs -0 -- ln -st /usr/local/bin \
```

Comment:
@fedde-s I would favor removing the find line if there is a cleaner way to import studies. We can update the import study with docker as well.

```ENTRYPOINT``` and ```CMD``` commands are very similar with the ```ENTRYPOINT``` simply requiring a different escape. 
- When running the container in detached (-d) the commands have no real difference.
- When running interactively with tty (-it) for testing or development, I would favor simplicity or allowing the user to build under different options i.e don't skip tests.

Following discussions in the past I suggest compiling cBioPortal before the ```CMD``` and ```ENTRYPOINT```. Such approach significantly limits the time it takes to run a docker container and allows the CMD to be limited to running apache tomcat.

Thoughts?